static/docs/commands-reference/version.md

```diff
@@ -0,0 +1,41 @@
+# version
+
+This command shows the system/environment information along with the DVC version.
+
+## Synopsis
+
+```usage
+    usage: dvc version [-h] [-q | -v]
+```
+
+## Options
+
+* `-h`, `--help` - prints the usage/help message, and exit.
+
+* `-q`, `--quiet` - does not write anything to standard output. Exit with 0 if
+  no problems arise, otherwise 1.
+
+* `-v`, `--verbose` - displays detailed tracing information.
+
+## Details
```

Comment:
this section should be `## Description` and it goes after `## Synopsis`

Reply:
@shcheklein 
So will `options` come after `description`?

Reply:
Yep, check this file https://dvc.org/doc/commands-reference/fetch, for example.